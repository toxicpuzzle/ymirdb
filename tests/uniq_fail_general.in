SET a 1 1 2 3 3 5 3 3 6
SET b 1
PUSH a b
UNIQ a
UNIQ b
LIST ENTRIES
bye