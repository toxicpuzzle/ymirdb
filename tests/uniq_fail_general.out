> ok

> ok

> ok

> simple entry only

> ok

> b [1]
a [b 1 1 2 3 3 5 3 3 6]

> bye
